debian/rdma-ndd.install

```diff
@@ -0,0 +1,4 @@
+etc/rdmam-ndd.conf
```

Comment:
As I did this I wondered if we should have a common config file...  But I need to investigate your systemd "environment" file stuff first.  When I first wrote this I did not include systemd support.  So the config file was the correct way to go.  Give me some time to investigate.